drivers/virtualbox/virtualbox.go

```diff
@@ -755,3 +775,35 @@ func getRandomIPinSubnet(baseIP net.IP) (net.IP, error) {
 
 	return dhcpAddr, nil
 }
+
+func (d *Driver) getLocalSSHClient() (ssh.Client, error) {
+	sshAuth := &ssh.Auth{
+		Passwords: []string{"docker"},
+		Keys:      []string{d.GetSSHKeyPath()},
+	}
+	sshClient, err := ssh.NewNativeClient(d.GetSSHUsername(), "127.0.0.1", d.SSHPort, sshAuth)
+	if err != nil {
+		return nil, err
+	}
+
+	return sshClient, nil
+}
+
+func translateWindowsMount(p string) (string, error) {
+	re := regexp.MustCompile(`(?P<drive>.+):\\(?P<path>.*)`)
```

Comment:
Do you think the `drive` here should at least be `[^:]+` (instead of the raw `.+` you've got here) to make sure we don't accidentally match past the first `:` to a second `:`?